pkg/compiler/gen.go

```diff
@@ -84,7 +84,9 @@ func (comp *compiler) collectCallArgSizes() map[string][]uint64 {
 				argPos[argID] = arg.Pos
 				continue
 			}
-			if argSizes[i] != typ.Size() {
+			if argSizes[i] != typ.Size() &&
+				arg.Pos.File != "sys/linux/auto.txt" && argPos[argID].File != "sys/linux/auto.txt" {
```

Comment:
I think it's better to looks for the auto attribute.
Call AST node has []Attrs slice.

Reply:
Done!